Song_sort_Genre.php

```diff
@@ -1,39 +1,70 @@
+<?php
```

Comment:
21/9: I add a function that deliver to the login page if they was login by want to access to some link that they should login first.